packages/test/test-end-to-end-tests/src/test/packageVersion.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Unlike normal packageVersion.ts, this is not generated.  Just import from package.json
```

Comment:
This is necessary just because this file isn't in the place that the generate script expects, I assume? Or is there another reason for this different approach?

Reply:
yeah....  this is probably temporary, since I do intent to move this in some other package.